docs/src/main/tut/datatypes/eithert.md

```diff
@@ -145,6 +145,19 @@ val myOptionET = EitherT.fromOption[Future](myOption, "option not defined")
 val myOptionListET = EitherT.fromOptionF(myOptionList, "option not defined")
 ```
 
+## From `ApplicativeError[F, E]` to `EitherT[F, E, A]`
+
+An `ApplicativeError[F, E]` or a `MonadError[F, E]`, can be converted into `EitherT[F, E, A]` 
+using the `attemptT` method on them.
+
+```tut:book
+val myTry: Try[Int] = Try(2)
+val myFuture: Future[String] = Future.failed(new Exception())
+
+val myTryET = myTry.attemptT
+val myFutureET = myFuture.attemptT
```

Comment:
Should we add type annotations to these as well to make it more obvious what the function does? :)

Reply:
I had a dilemma with this, but the tut:book creates a kinda great output so I let it go...
Probably using tut:silent and type annotations would be an another good option.

Reply:
So what do you think? Should I add the return types, or let the `tut:book` to do its magic?

Reply:
IMO I think it's better to add the type annotations, just to be fully clear even in the original form. Also at some point I'm assuming we'll be switching from tut to mdoc, so it's worth thinking a bit ahead.